packages/augur-ui/src/modules/common/form.tsx

```diff
@@ -1765,14 +1781,17 @@ export class Input extends Component<InputProps, InputState> {
     window.addEventListener('click', this.handleWindowOnClick);
   }
 
-  UNSAFE_componentWillReceiveProps(nextProps) {
-    const { value } = this.props;
+  static getDerivedStateFromProps(nextProps: InputProps, state: InputState) {
+    const { value } = state;
     if (value !== nextProps.value) {
-      this.setState({ value: nextProps.value });
+      return { value: nextProps.value };
+    }
+    else {
+      return null
     }
   }
 
-  UNSAFE_componentWillUpdate(nextProps, nextState) {
+  componentDidUpdate(prevProps, prevState) {
     if (
       nextProps.canToggleVisibility &&
```

Comment:
nextProps is no longer defined. Just bring up the Ui would have found this in about 1 second